litellm/model_prices_and_context_window_backup.json

```diff
@@ -591,15 +591,36 @@
     },
     "mistral/mistral-small": {
         "max_tokens": 8192,
-        "input_cost_per_token": 0.00000066,
-        "output_cost_per_token": 0.00000197,
+        "input_cost_per_token": 0.000002,
+        "output_cost_per_token": 0.000006,
```

Comment:
this was indeed the price at one point (jan -> https://web.archive.org/web/20240103230150/https://docs.mistral.ai/platform/pricing/)
but docs now state that the small model pricing has changed -> https://docs.mistral.ai/platform/pricing/